src/ipld/util.rs

```diff
@@ -468,6 +372,11 @@ impl<DB, T> UnorderedChainStream<DB, T> {
             Err(v) => v.lock().clone(),
         }
     }
+
+    #[allow(dead_code)]
+    pub async fn join_workers(self) -> anyhow::Result<()> {
+        self.worker_handle.await?
+    }
```

Comment:
Why do we need dead code?

Reply:
This is needed for the parallel graph traversal. That is being debugged at the moment.

Reply:
fixed